.readthedocs.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2017 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+version: 2
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.10"
+
+python:
+  install:
+    - requirements: docs/requirements.txt
```

Comment:
Was this file generated? Can you add a comment stating which command was used to generate the file?

Reply:
For sure :) I was thinking of adding the command also to the `make docs` make instructions.

Reply:
I did add it to `make docs` and created a separate `make docs-ci` to only run the sphinx build command when testing the generation process in the CI workflow.